### DISCUSSION TITLE
专业版显示1970年1月1日到期

### DISCUSSION THREAD

**Discussion by mmk126** [getlantern/lantern-forum-cn#784] [Q&A]
前一天刚续费1年，今天页面显示1970年1月1日到期。非常慢，是不是变成免费版了？
附图我的续费记录以及后台显示。我已经用了8年了，真的越来越慢，续费也只敢一年一年续，就怕哪天你们甩手不干了。哎。。。
<img width="347" alt="Screenshot 2024-08-14 at 22 07 24" src="https://github.com/user-attachments/assets/b5efb0d3-cad5-440b-8684-820f4b98863b">
![IMG_8240](https://github.com/user-attachments/assets/65675f1a-dd11-4d4d-9bb8-1416b94902f6)

**Comment by MaxMKMax** [comment]
@mmk126 您好. 谢谢您的反馈! 好的, 我已经帮您提交问题上去了. 可能变免费了所以变慢. 请您提供我您的账号邮箱. 
您放心吧, 我们不可能会不做的. 一直提供你们服务是我们的终极目标!

**Reply by MaxMKMax** [reply]
@mmk126 正在修复了. 但好像又有另一个bug了. 我来反馈一下让技术团队紧急修复.

**Reply by JMaster1937** [reply]
我的问题为什么不给解决！

**Comment by MaxMKMax** [comment]
@mmk126 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by mmk126** [reply]
ticket #75876541

**Reply by mmk126** [reply]
ticket#76105505, 又是一样的问题，user data not found. pro account 用不了。

**Reply by mmk126** [reply]
ticket 76181412, 又变免费版了

**Comment by MaxMKMax** [comment]
@mmk126 好的. 我幫您提交問題上去.

**Comment by MaxMKMax** [comment]
@mmk126 
技术人员的回复:
我在 JitBit 中看到他们正在运行 v7.9.1，因此我建议更新到 v7.9.4 以查看是否可以清除错误。
